invehicle-apps/kuksa-traccar-client/providers/kuksa/clientComm.py

```diff
@@ -0,0 +1,74 @@
+#! /usr/bin/python3
```

Comment:
@SebastianSchildt This file is duplicated to https://github.com/eclipse/kuksa.val/blob/master/vss-testclient/clientComm.py. What should we do?

Reply:
create issue https://github.com/eclipse/kuksa.val/issues/111 to solve the problem